register-service-broker.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+set -u
+
```

Comment:
You may want to do some validation of your env vars here before using them below. What happens if you have none set etc?